experimental/C/src/RecursiveInstantiation/SieveOfErastothenesUnrolled.lf

```diff
@@ -0,0 +1,362 @@
+/**
+ * This is an expanded version of SieveOfErastothenes.lf. It duplicates
+ * code to prevent recursive instantiation, which is currently forbidden.
```

Comment:
Perhaps a pointer to the version that does not compile would be good?  With text like "It would be better to implement this using recursive instantiation, as attempted in <other file>, but recursive instantiation is currently not supported."